src/engine/ruleset/decoders/windows/windows-sysmon.yml

```diff
@@ -0,0 +1,205 @@
+name: decoder/windows-sysmon/0
+
+sources:
+  - decoder/windows-event-decoder/0
+
+metadata:
+  description: Decoder for Windows Sysmon events
+  module: Windows Sysmon
+  title: Decoder for Windows Sysmon
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.3
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/23
+  references:
+    - https://learn.microsoft.com/es-es/sysinternals/downloads/sysmon
+
+check:
+  - ~windows.Event.System.Provider.@Name: Microsoft-Windows-Sysmon
+
+
+normalize:
+ - map:
+    - event.module: sysmon
+    - event.kind: event
+
+    # Add categorization fields
+    - event:  +kvdb_get_merge/sysmon_categories/$event.code
+
+    - rule.name: $~windows.Event.EventData.RuleName
+    - event.hash: $~windows.Event.EventData.Hash
+    - ~temp.hashes: +parse_kv/$~windows.Event.EventData.Hashes/=/,/'/'
```

Comment:
Due to renaming in helpers functions (issue recently merged on 11334-dev-new-wazuh-engine)
I'm having trouble trying to validate the decoders:
```
╰─# ./main catalog  validate decoder/windows-sysmon/0 < /home/vagrant/workspace/wazuh/src/engine/ruleset/decoders/windows/windows-sysmon.yml         
Stage block "map" building failed: An error occurred while building the helper function "parse_kv": Builder name "helper.parse_kv" could not be found in the registry
```

Reply:
Done